Source/PagesController.swift

```diff
@@ -177,8 +177,10 @@ extension PagesController {
   }
 
   private func toggle() {
-    for recognizer in gestureRecognizers {
-      (recognizer as! UIGestureRecognizer).enabled = enableSwipe
+    for view in self.view.subviews {
```

Comment:
@richardoti We could skip `self` because of https://github.com/hyperoslo/iOS-playbook/blob/master/style-guidelines/Swift.md#use-of-self